src/Transports.AspNetCore.NewtonsoftJson/Transports.AspNetCore.NewtonsoftJson.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
In this project we need to add an extension method for Newtonsoft.Json integration to conveniently connect all the necessary components with IntelliSense help:
```c#
 public static class GraphQLBuilderExtensions
    {
        public static IGraphQLBuilder AddNewtonsoftJson(this IGraphQLBuilder builder, Action<JsonSerializerOptions> configure = null)
        {
            builder.Services.AddSingleton<IGraphQLRequestDeserializer>(p => new GraphQLRequestDeserializer(configure ?? _ => { }))
            builder.Services.AddSingleton<IDocumentWriter, DocumentWriter>()

            return builder;
        }
}
```

The same for S.T.J.

Reply:
Done, also needed to support configuration of the `DocumentWriter`, so added a param for that too.

Also covered readme.